addons/sql_shell/models/sql_shell.py

```diff
@@ -0,0 +1,50 @@
+from psycopg2 import Error as Psycopg2Error
+import json
+
+from odoo import models, fields, api
+
+class SQLShell(models.Model):
```

Comment:
Naive question : Is there a reason to not use classic Transient Model like other wizard ?